examples/.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["../babel"]
+  "presets": ["../packages/next/babel"]
```

Comment:
Can you remove the file, it's not needed anymore.

Reply:
Yep 👍

I also removed examples/.gitignore because it's also not needed anymore.